translations/README-uz.md

```diff
@@ -185,30 +185,35 @@ If you want to be a reliability engineer or operations engineer, study more from
 
 ---
 
-## Why use it?
+## Nima uchun foydali?
 
-When I started this project, I didn't know a stack from a heap, didn't know Big-O anything, anything about trees, or how to
-traverse a graph. If I had to code a sorting algorithm, I can tell ya it wouldn't have been very good.
-Every data structure I've ever used was built into the language, and I didn't know how they worked
-under the hood at all. I've never had to manage memory unless a process I was running would give an "out of
-memory" error, and then I'd have to find a workaround. I've used a few multidimensional arrays in my life and
-thousands of associative arrays, but I've never created data structures from scratch.
+Ushbu loyihani boshlaganimda, men bu stack dagi to’plamlar(heap) haqida ham,
+Big-O, daraxtlar(trees) haqida va grafikni qanday bosib o'tish kerakligi haqida hech narsa 
+bilmas edim. Agar tartiblash algoritmida kod yozishga kelsak bu ham menda juda yaxshi bo’lmagan.  
+Men ilgari dasturlash tilga kiritilgan ma'lumotlar tuzilmasini ishlatganimda uning asl mohiyatini 
+va qanday ishlashlarini umuman bilmas edim. Ish jarayonida agar "xotiradan tashqarida" xatolikka
+yo'l qo'yilmasa, men xotirani boshqarish haqida qayg’urmaganman  va xatolikka duch kelsam ham, 
+vaqtinchalik yechim topishga to'g'ri kelgan. Men hayotimda bir nechta ko'p 
+o'lchovli massivlardan((multidimensional arrays)) 
+va minglab assotsiativ massivlardan foydalanganman, lekin men hech qachon noldan ma'lumotlar 
+tuzilmalarini yaratmaganman.
```

Comment:
"Ushbu loyihani boshlaganimda, men stack bilan heap-ni farqini bilmas edim,
Big-O ni nimaligini bilmas edim, tree-lar yoki graph-lar haqida hech qanday tushunchaga ega emas edim. Agar tartiblash algoritmini (sorting) yozishni surashganda, aniq qovun tushurgan bular edim. Umrimda ishlatgan data strukturalarni hammasini tayyor xolda ishlatganman (built-in library) va ularni asosida nimalar yotishini umuman bilmas edim. Juda ko'p kod yozgan bulsamda, lekin men hech qachon qandaydir data strukturani noldan yozmaganman."

Iltimos tepadagini kurib chiqib, sizga maqul bulsa shunga uzgartiraylik. Agar e'tibor bergan bulsez, men hamma narsani ham tugridan tugri tarjima qilmadim va bazi narsalarni tashlab kettim. Bizni asosiy maqsadimiz, manoni yetkazib berish. Google translate yaramas ekan.

Shunday tarizda davom etaylik :)

Reply:
replace qildim